babel.config.js

```diff
@@ -1,5 +1,22 @@
-const path = require('path');
-
 module.exports = {
-  extends: path.resolve(process.cwd(), './node_modules/carbon-factory/babel.config.js')
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'usage',
+        targets: {
+          browsers: 'IE 11'
```

Comment:
Since we're dropping IE11 support going forwards - do we need this?

Reply:
Good point will remove

Reply:
Good point I'll remove